src/js/view/view.js

```diff
@@ -928,6 +934,18 @@ define([
             _controlsTimeout = setTimeout(_userInactive, _timeoutDuration);
         }
 
+        function _toggleControls() {
```

Comment:
@pajong this would cause the player to override the `"controls": false` setting in the config, which we don't want to do.